source/api_documentation/network_integration/advertiser_campaigns/index.rst

```diff
@@ -211,10 +223,6 @@ You are not allowed to delete campaigns.
     - integer
     - Default Creative ID.
 
-  * - max_promo_numbers
-    - integer
-    - Maximum Promo Numbers.
-
```

Comment:
Removed this attribute because it's not to be found in the response. It was removed more than a year ago from the dto.